assets/scripts/main.js

```diff
@@ -3,47 +3,32 @@
  *
  */
 
-// Initalization scripts
-import './app/initialization'
+import $ from 'jquery'
+window.$ = $
 
-// Export modules to window for the global-scoped JavaScript
-import './exports'
+import _ from 'lodash'
+window._ = _
 
-// Load things that are needed
-import './app/event_tracking'
+import { Stmx } from './app/initialization'
+window.Stmx = Stmx
+
+// import modules for side-effects
 import './app/blocking_shield'
 import './app/debug_info'
 import './app/keyboard_commands'
 import './app/print'
 import './app/status_message'
 import './app/welcome'
-
-// Menus
-import './menus/menu'
-import './menus/_feedback'
-import './menus/_help'
-import './menus/_identity'
```

Comment:
Similar to the "Save as image" one, the side effect attaches an event listener so that clicking on your username opens a menu.